rplugin/python3/denite/source/man.py

```diff
@@ -4,27 +4,39 @@
 
 
 class Source(Base):
+    SEARCH_TYPE_SWITCH = {
+        'desc' : '--apropos',
+        'name' : '--whatis',
+    }
 
     def __init__(self, vim):
         super().__init__(vim)
 
         self.name = 'man'
         self.kind = 'man'
 
-        self.__cmd = ['man', '-k']
-
     def gather_candidates(self, context):
         word = context['input']
 
         if len(word) == 0:
-            word = '.'
+            word = self._get_arg( context, 0, default='.' )
+
+        sections = self._get_arg( context, 1, default=None )
+
+        if word != '.':
+            search_type = self._get_arg( context, 2, default='desc' )
```

Comment:
Please remove the whitespace around the parentheses.

Reply:
Fixed